### PR TITLE
Updated @sentry dependencies and bumped version.

### PR DESCRIPTION
**Pull request by schmitzt3**

### CODE REVIEW

**Comment by schmitzt3** [comment]
This should also fix some issues that people will see with Ionic 4. Let me know if I need to do anything else to get this PR through.

Thanks for your contributions!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sentry-cordova",
-  "version": "0.13.1",
+  "version": "0.14.1",
```
Comment (HazAT) [inline package.json]: Please do not bump the version, we will do this when releasing.

File: package.json [hunk 2]
```diff
@@ -24,11 +24,11 @@
   ],
   "description": "Official Sentry SDK for Cordova",
   "dependencies": {
-    "@sentry/browser": "~4.3.0",
-    "@sentry/core": "~4.3.0",
-    "@sentry/minimal": "~4.3.0",
-    "@sentry/types": "~4.3.0",
-    "@sentry/utils": "~4.3.0",
+    "@sentry/browser": "~4.6.0",
```
Comment (HazAT) [inline package.json]:
```suggestion
    "@sentry/browser": "~4.6.6",
```

File: package.json [hunk 3]
```diff
@@ -24,11 +24,11 @@
...
+    "@sentry/core": "~4.6.0",
```
Comment (HazAT) [inline package.json]:
```suggestion
    "@sentry/core": "~4.6.6",
```

File: package.json [hunk 4]
```diff
@@ -24,11 +24,11 @@
...
+    "@sentry/minimal": "~4.6.0",
```
Comment (HazAT) [inline package.json]:
```suggestion
    "@sentry/minimal": "~4.6.5",
```

File: package.json [hunk 5]
```diff
@@ -24,11 +24,11 @@
...
+    "@sentry/types": "~4.5.0",
```
Comment (HazAT) [inline package.json]:
```suggestion
    "@sentry/types": "~4.5.3",
```

File: package.json [hunk 6]
```diff
@@ -24,11 +24,11 @@
...
+    "@sentry/utils": "~4.6.0",
```
Comment (HazAT) [inline package.json]:
```suggestion
    "@sentry/utils": "~4.6.5",
```

**Comment by schmitzt3** [comment]
> Thanks for the PR, some minor things.
> Also did you try it out with ionic 4 and confirms it fixes it?

I did test this with Ionic 4 by issuing the following command:

`npm install --save https://github.com/schmitzt3/sentry-cordova`

After updating the packages and using my fork, I was able to get sentry to work properly. With Ionic 4 there was an out of memory exception when serializing the exception. However, when running on a physical iOS device, this plugin seemed to slow down my app.

Would it be helpful for me to create a skeleton app with only Ionic 4 basics and this plugin?

**Comment by HazAT** [comment]
@schmitzt3 I think it's fine thanks, I will merge this and make a release.

**Comment by schmitzt3** [comment]
Thanks @HazAT!

**Pull request merged**